flow_screen_components/richTextInputFSC/force-app/main/default/lwc/inputRichTextFSC/inputRichTextFSC.js

```diff
@@ -1,9 +1,215 @@
-import { LightningElement, api } from 'lwc';
+import { LightningElement, api, track } from 'lwc';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
 
 export default class InputRichTextFSC_LWC extends LightningElement {
+
+    //Input and Output Attributes for Flow
     @api value;
+    @api richerText = false;
+    @api disallowedWordsList;
+    @api disallowedSymbolsList;
+    @api autoReplaceMap;
+    @api warnOnly = false;
     @api label;
     formats = ['font', 'size', 'bold', 'italic', 'underline',
         'strike', 'list', 'indent', 'align', 'link',
         'image', 'clean', 'table', 'header', 'color','background','code','code-block','script','blockquote','direction'];
-}
\ No newline at end of file
+
+    //Validation hook to use standard in Flow.  Only enforce if 
+    @api validate(){
+        this.value = this.richText;
+        if(!this.richerText || this.warnOnly || (!this.warnOnly && this.isValidCheck)){
+            return {isValid:true};
+        }else{
+            return {
+                isValid:false,
+                errorMessage: 'Cannot Advance - Invalid Symbols/Words Remaind in Rich Text'
```

Comment:
it might be hard to figure out what's causing the problem. think about returning the disallowed hits in this error string.

Reply:
Great - incorporated this into latest version.